src/js/mixins/TooltipMixin.jsx

```diff
@@ -38,7 +38,8 @@ var TooltipMixin = {
   tip_handleContainerMouseMove: function (e) {
     var el = e.target;
 
-    if (el.dataset && el.dataset.behavior && el.dataset.behavior === "show-tip") {
+    if (el.dataset && el.dataset.behavior
+        && el.dataset.behavior === "show-tip") {
```

Comment:
OK we can take the discussion out of this PR. https://github.com/mesosphere/marathon/issues/2163